app/src/main/java/com/example/goship/ui/orders/DataManager.kt

```diff
@@ -1,50 +1,50 @@
 package com.example.goship.ui.orders
 
-data class CustomerOrders (val orderId: Int, val u_email: String, val status: String) {
+data class Orders (val orderId: Int, val u_email: String, val price: String, val p_date: String) {
 //    override fun toString(): String {
 //        return cityName
 //    }
 }
 
 //Singleton declaration of the data top populate the spinner and tables
 object DataManager {
-    val customerOrders = HashMap<Int, CustomerOrders>()
+    val customerOrders = HashMap<Int, Orders>()
 
     init {
         initializeCustomerOrders()
     }
 
     private fun initializeCustomerOrders() {
         var order =
-            CustomerOrders(5392170, "me@yahoo.com", "closed")     //create the city
+            Orders(5392170, "me@yahoo.com", "185.32", "04/05/2020")     //create the city
```

Comment:
I'd be more comfortable if you add a loop with data first and then pass a list with this dummy instead of this. It'll be easier later when we change code.